src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAO.scala

```diff
@@ -533,12 +533,12 @@ class PostgresAccessPolicyDAO(protected val dbRef: DbReference,
                and ${p.public} = true"""
 
     import SamTypeBinders._
-    runInTransaction { implicit session =>
+    runInTransaction("listPublicAccessPolicies", samRequestContext)({ implicit session =>
```

Comment:
can we differentiate the names of overloaded functions? There are multiple 'listPublicAccessPolicies' and we should be able to see them differently in traces. (Are there other overloaded functions?)

Reply:
we can un-overload them if need be

Reply:
Good catch -- I didn't notice that we had an overloaded function here. I can either rename/un-overload the functions or change the name that we pass to the trace. Any reason to go through the additional effort of the former over the latter?

Reply:
just changing the trace name sounds fine to me and I don't think we're trying to forbid overloads.